src/modules/poweraccent/PowerAccentKeyboardService/KeyboardListener.idl

```diff
@@ -60,6 +60,8 @@ namespace PowerToys
         {
             None,
             Space,
+			Left,
+			Right,
```

Comment:
Please use spaces instead of tabs to keep the file consistent.

Reply:
Done.